src/tests/ftest/osa/osa_online_reintegration.py

```diff
@@ -222,22 +222,37 @@ def run_online_reintegration_test(self, num_pool):
             self.pool.display_pool_daos_space("Pool space: Beginning")
             pver_begin = self.get_pool_version()
             self.log.info("Pool Version at the beginning %s", pver_begin)
-            output = self.dmg_command.pool_exclude(self.pool.uuid,
-                                                   rank, t_string)
+            if server_boot is False:
+                output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                       rank, t_string)
+            else:
+                output = self.dmg_commmand.system_stop(self.pool.uuid,
+                                                       rank)
             self.log.info(output)
-
             fail_count = 0
             while fail_count <= 20:
                 pver_exclude = self.get_pool_version()
                 time.sleep(10)
                 fail_count += 1
                 if pver_exclude > (pver_begin + len(target_list)):
                     break
+            if server_boot is False:
+                self.log.info("Pool Version after exclude %s", pver_exclude)
+                # Check pool version incremented after pool exclude
+                self.assertTrue(pver_exclude > (pver_begin + len(target_list)),
+                                "Pool Version Error:  After exclude")
+            else:
+                self.log.info("Pool Version after server stop %s",
+                              pver_exclude)
+                # Check pool version incremented after pool exclude
+                self.assertTrue(pver_exclude > (pver_begin + len(target_list)),
+                                "Pool Version Error:  After server stop")
+                # Now start the server
+                output = self.dmg_commmand.system_start(self.pool.uuid,
```

Comment:
(pylint-no-member) Instance of 'OSAOnlineReintegration' has no 'dmg_commmand' member